test/api-digester/Outputs/stability-stdlib-abi.swift.expected

```diff
@@ -483,6 +483,23 @@ Protocol _NSEnumerator has been removed
 Protocol _NSFastEnumeration has been removed
 Protocol _ShadowProtocol has been removed
 
+Constructor _StringGutsSlice.init(_:) has been removed
+Constructor _StringGutsSlice.init(_:_:) has been removed
+Func _StringGutsSlice._normalizedHash(into:) has been removed
+Func _StringGutsSlice.compare(with:expecting:) has been removed
+Func _StringGutsSlice.withFastUTF8(_:) has been removed
+Struct _StringGutsSlice is now without @_fixed_layout
```

Comment:
This is suspicious; it should just say "Struct _StringGutsSlice has been removed".

Reply:
Yep. Could be an API digester bug with handling of @usableFromInline, since the declaration is internal.